tensilelite/Tensile/SolutionStructs.py

```diff
@@ -1956,6 +1956,26 @@ def isDirectToVgprDoable(state, tc):
       reject(state, "DirectToVgpr%c does not supports Sparse"%(tc))
       return False
 
+    # for DTVA, does not work with PGR0
+    if tc == 'A' and state["PrefetchGlobalRead"] == 0:
```

Comment:
is DTVB ok?

Reply:
I can have a test since that ticket is reported by me. We are currently focusing more on DTVA, some DTVB cases are additionally tested.

Reply:
@geotseng-amd I've updated the ticket, so you can directly reject PGR=0 (A/B) have same failure